CONTRIBUTING.md

```diff
@@ -1,53 +1,307 @@
+Labels related to [issue automation](#appendix-issue-automation)
+
+- [Needs attention](https://github.com/OAI/OpenAPI-Specification/labels/Needs%20attention) automated tag when an issue is updated
+- [Needs author feedback](https://github.com/OAI/OpenAPI-Specification/labels/Needs%20author%20feedback) used to indicate that more information is needed from the issue creator
+- [No recent activity](https://github.com/OAI/OpenAPI-Specification/labels/No%20recent%20activity) if no information is received, the issue is marked for closure (automatic after 30 days)
+
+### Milestones
+
+We use milestones in GitHub to plan what should be included in future releases.
+Issues and pull requests should both be added to the milestone we expect they will be released in.
+Any changes that aren't ready in time for release should be moved to the next milestone or untagged.
+
+The milestones and items assigned to them are under constant review and subject to change.
+
+### Projects
+
+The OpenAPI Initiative uses GitHub Projects to manage work _outside_ of the specification development process.  There are currently two active projects:
+
+* [Contributor Guidance](https://github.com/orgs/OAI/projects/5/views/1)
+* [Automation & Infrastructure](https://github.com/orgs/OAI/projects/4/views/1)
+
+## Pull requests
+
+> [!NOTE]
+> Since the 3.0.4 and 3.1.1 releases (October 2024), the OAS is developed in the `src/oas.md` file.
+> Check the [Branches](#branches) section for more information about the updated branching strategy.
+
+Changes to the next version of the specification are welcome and can be proposed by anyone.
+
+For large changes that will need discussion, please use the [Proposal process](#propose-a-specification-change).
+For other changes, we recommend [opening an issue](#issues) first, so that you can get some feedback and any extra input you need before spending a lot of time on something.
+
+Schema changes are made on the same branch, but can be released independently.
+When making a specification change for a new minor or major release that has a schema impact, including the schema change in the PR is preferred.
+Patch releases cannot contain changes that _require_ a schema update.
+
+### Use a fork
+
+All work **MUST be done on a fork** and be submitted as a pull request.
+
+### Target the earliest active `*-dev` branch
+
+Branch from and submit pull requests to the a branch from the _earliest relevant and [active](#active-branches)_ `vX.Y-dev` branch.
+For example, if a change applies to both 3.1 and 3.2, the PR would go to the `v3.1-dev` branch, which will be merged up to `v3.2-dev` before the next 3.2 release.
+All changes to the specification must conform to the [style guide](./style-guide.md).
+
+Both specification and schema changes follow this approach.
+
+For changes to repository files that affect all versions, use the `main` branch.
+This might apply to, for example, Markdown files, automation, and scripts.
```

Comment:
There might be things we want to start on `dev`, but I'm just saying that to make a note of it- I don't think there is anything to say about it yet.  I think we just need to figure out a bit more what counts as infrastructure that is not needed on `main` (and shouldn't clutter it up) vs information that should be on `main` as the most public and obvious place to find things ilke this document.